### ISSUE TITLE
Unable to upgrade Allor

### ISSUE THREAD

**Issue by lunatico67** [Nourepide/ComfyUI-Allor#23]
Unable to upgrade Allor
![Allor](https://github.com/Nourepide/ComfyUI-Allor/assets/79542282/59b390c7-9767-4abc-bb8b-f42a4521345b)

**Comment by assassindesign** [comment]
Encountered the same problem, the plug-in cannot be recognized and used

**Comment by Nourepide** [comment]
The current message doesn't tell me anything.

Please show me the error from the terminal.

**Comment by lunatico67** [comment]
date custom node 'Allor Plugin'
Update: ['https://github.com/Nourepide/ComfyUI-Allor']
Cmd('git') failed due to: exit code(128)
  cmdline: git pull -v -- origin
  stderr: 'fatal: refusing to merge unrelated histories'
CUSTOM NODE PULL: Fail
Update was successful.
After restarting ComfyUI, please refresh the browser.
FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json

**Comment by Nourepide** [comment]
Thanks, understood. This kind of error can be easily fixed.

Here's some information on why [it happened](https://github.com/Nourepide/ComfyUI-Allor?tab=readme-ov-file#image-rebasing).

The easiest way for you to fix it, go to the `custom_nodes` folder and delete there the folder with the Allor plugin and then reinstall it.

**Comment by lunatico67** [comment]
Hi
Thanks for the help but I have deleted and reinstalled it and it still does not work.

![Allor2](https://github.com/Nourepide/ComfyUI-Allor/assets/79542282/e671de77-6b76-4114-aa00-99f41e33384d)

When i load the workflow  Nerd-Reposer Plus


![Allor3](https://github.com/Nourepide/ComfyUI-Allor/assets/79542282/c0fc2b66-09a1-4c8f-a99d-4df6e3638ea5)

**Comment by lunatico67** [comment]
> Encountered the same problem, the plug-in cannot be recognized and used

Did you manage to solve it?

**Comment by assassindesign** [comment]
There is still an error. I raised the same issue in comfyui. It seems to be an error in the plug-in itself.

**Comment by lunatico67** [comment]
> There is still an error. I raised the same issue in comfyui. It seems to be an error in the plug-in itself.

Yes, with the latest Comfyui´s updates Allor  stopped working. Hopefully they will fix it soon because there are many good templates that require it.

**Comment by Nourepide** [comment]
Okay, guys, first of all, happy new year to you.
Secondly I got very unlucky sick right after the new year, but don't worry, even during my halluciogenic dreams at 40 degrees Celsius I was still thinking about this problem. :D

So, what I see is very non-standard, to be more precise it looks like the manager is loading an outdated version of Allor after reinstallation.
No need to panic, we have at least two more options to fix it.

The first option is to let Allor handle the problem itself (if your version of Allor is not quite old). Simply configure your config.json [according to these guidelines](https://nourepide.github.io/ComfyUI-Allor-Doc/updating.html#troubleshooting_auto_update).

Second option, if you have Git installed you can manually update Allor using [these instructions](https://nourepide.github.io/ComfyUI-Allor-Doc/updating.html#troubleshooting_git) as well.

> are many good templates that require it

You'd know how nice it is to read something like that.
